exercises/practice/etl/.meta/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "authors": [
+    "mfoda"
+  ],
+  "files": {
+    "solution": [
+      "src/Etl.hx"
```

Comment:
@BNAndras Apparently this solution file in the config and the actual name of the file need to match.

Previously we had named the file `ETL.hx` with the config looking for `Etl.hx`, but the `bin/build.sh` script enters the `.meta` directory, copies `Example.hx` file to a new filename: the PascalCase'd version of the exercise slug. Sooooo, the file being used by the test is `Etl.hx` no matter what we do, so the only "solution" is to commit to the PascalCase.

In order to fix this, we'd need to change how the build script works.😅